<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plotCanvas</class>
 <widget class="QWidget" name="plotCanvas">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>989</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1064</width>
    <height>989</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1064</width>
    <height>989</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Differential Expression Analysis</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>590</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Statistics Report</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>620</y>
     <width>210</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Genes in Selection A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>650</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Genes in Selection B</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numGenesSelectionA">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>620</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="numGenesSelectionB">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>650</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="meanSelectionB">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>710</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>680</y>
     <width>230</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mean of Distribution in Selection A</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="meanSelectionA">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>680</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>710</y>
     <width>220</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mean of Distribution in Selection B</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stdDevSelectionA">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>740</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="stdDevSelectionB">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>770</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>740</y>
     <width>270</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Standard Dev of Distribution in Selection A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>770</y>
     <width>280</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Standard Dev of Distribution in Selection B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>800</y>
     <width>220</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Correlation between distributions</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="correlation">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>800</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>830</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Overlapping Genes</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="overlappingGenes">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>830</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>860</y>
     <width>300</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Genes only present in Selection A</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="genesOnlyA">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>860</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="genesOnlyB">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>890</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>890</y>
     <width>290</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Genes only present in Selection B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>520</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Selection A </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>550</y>
     <width>219</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Selection B</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="selectionA">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>520</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="selectionB">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>10</y>
     <width>500</width>
     <height>900</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>900</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>900</height>
    </size>
   </property>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QWidget" name="plotWidget" native="true">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>13</y>
     <width>521</width>
     <height>491</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>521</width>
     <height>491</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>521</width>
     <height>491</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>838</x>
     <y>930</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Save the plot to a file</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>958</x>
     <y>930</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Closes the plot</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
